code/datums/dna.dm

```diff
@@ -6,6 +6,7 @@
 	var/blood_type
 	var/datum/species/species = new /datum/species/human //The type of mutant race the player is if applicable (i.e. potato-man)
 	var/list/features = list("FFF") //first value is mutant color
+	var/uni_features
```

Comment:
What is `uni`?
Document this and give it a clear name.
The code around this is old, so don't use it as an example for your own.